CustomLookup/CustomLookup.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="1.0.29" />
+    <PackageReference Include="Moq" Version="4.12.0" />
```

Comment:
Why is the main project, not the test project, referencing Moq?

Reply:
Actually, it seems like your tests aren't using Moq at all, so you should be able to remove this entirely.